### DISCUSSION TITLE
How to access not translated cell?

### DISCUSSION THREAD

**Discussion by mohjacksi** [spatie/laravel-translatable#337] [Q&A] [answered]
Hello, I'm integrating Laravel-Translatable with an existing Laravel project of mine, I'm facing a problem here, when retrieve the data, I cant get the original content of it which it not translated yet! (didn't converted to json yet!)
What I want is accessing the untranslated data when it's available and show the translated data when available.
It's very important for me to show untranslated values as normal as it was!
Like I have 'name' not translated I have to access it like
$person->name; //Mohammed (not translated)
$person->name; //محمد (translated and the local is Arabic) 
$person->name; //Mohammed (translated and  the local is English)

**Comment by anibalealvarezs** [comment, marked as answer]
Hello.

Check this response in this other issue: https://github.com/spatie/laravel-translatable/issues/320#issuecomment-1017410024

**Reply by mohjacksi** [reply]
Thank you @anibalealvarezs, I solve it like this:
```
public function getNameAttribute($value)
    {
        // if the value has no translation 
        if($value == '') 
            return $this->getRawOriginal('name');

        return $value;
    }
```
